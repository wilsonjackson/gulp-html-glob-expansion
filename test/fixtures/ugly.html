<html>
<head>
	<title>You Always Look Not That Fine</title>

      <SCRIPT SRC="/assets/*.js"></script> <script src = /assets/*.js></script><script src='/assets/file.js' type="text/javascript"></script>

<link href = /assets/otherfile.css >
 		 <LINK href="/assets/*.css" rel="stylesheet" TYPE=text/css>
</head>
</html>
