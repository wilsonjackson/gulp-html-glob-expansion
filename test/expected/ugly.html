<html>
<head>
	<title>You Always Look Not That Fine</title>

      <SCRIPT SRC="/assets/file.js"></script>
      <SCRIPT SRC="/assets/otherfile.js"></script> <script src = /assets/file.js></script>
 <script src = /assets/otherfile.js></script><script src='/assets/file.js' type="text/javascript"></script>

<link href = /assets/otherfile.css >
 		 <LINK href="/assets/file.css" rel="stylesheet" TYPE=text/css>
 		 <LINK href="/assets/otherfile.css" rel="stylesheet" TYPE=text/css>
</head>
</html>
